src/main/java/org/minions/devfund/angela/battleship/Ocean.java

```diff
@@ -0,0 +1,192 @@
+        shotsFired = 0;
+        hitCount = 0;
+        shipsSunk = 0;
+        for (int i = 0; i < OCEAN_SIZE; i++) {
+            for (int j = 0; j < OCEAN_SIZE; j++) {
+                EmptySea emptySea = new EmptySea();
+                emptySea.placeShipAt(i, j, true, this);
+            }
+        }
+    }
+
+    /**
+     * Places 13 ships randomly.
+     */
+    void placeAllShipsRandomly() {
+        Random random = new Random();
+        List<Ship> shipsToPlace = Arrays.asList(new BattleShip(), new BattleCruiser(), new Cruiser(), new Cruiser(),
+                new LightCruiser(), new LightCruiser(), new Destroyer(), new Destroyer(), new Destroyer(),
+                new Submarine(), new Submarine(), new Submarine(), new Submarine());
+
+        for (Ship ship : shipsToPlace) {
+            boolean located = false;
+            while (!located) {
+                int row = random.nextInt(OCEAN_SIZE);
+                int column = random.nextInt(OCEAN_SIZE);
+                boolean horizontal = random.nextBoolean();
+                if (ship.okToPlaceShipAt(row, column, horizontal, this)) {
+                    ship.placeShipAt(row, column, horizontal, this);
+                    located = true;
+                }
+            }
+        }
+    }
+
+    /**
+     * Verifies if position was occupied.
+     *
+     * @param row    int row position.
+     * @param column int column position.
+     * @return true if position was occupied.
+     */
+    boolean isOccupied(final int row, final int column) {
+        return ships[row][column].getLength() > 1;
+    }
+
+    /**
+     * Shots a position.
+     *
+     * @param row    int row position.
+     * @param column int column position.
+     * @return true if ship was shot.
+     */
+    boolean shootAt(final int row, final int column) {
+        shotsFired++;
+        if (ships[row][column].shootAt(row, column)) {
+            if (ships[row][column].isSunk()) {
+                shipsSunk++;
+            }
+            hitCount++;
+            return true;
+        }
+        return false;
+    }
+
+    /**
+     * Obtains the shots fired.
+     *
+     * @return int shots fired quantity.
+     */
+    int getShotsFired() {
+        return shotsFired;
+    }
+
+    /**
+     * Obtains the hit count.
+     *
+     * @return int he hit number.
+     */
+    int getHitCount() {
+        return hitCount;
+    }
+
+    /**
+     * Obtains the ships sunk quantity.
+     *
+     * @return int ships sunk number.
+     */
+    int getShipsSunk() {
+        return shipsSunk;
+    }
+
+    /**
+     * Verifies if is game over.
+     *
+     * @return true if game over.
+     */
+    boolean isGameOver() {
+        return shipsSunk == SHIPS_QUANTITY;
+    }
+
+    /**
+     * Gets the ship array.
+     *
+     * @return matrix of {@link Ship}
+     */
+    Ship[][] getShipArray() {
+        return ships;
+    }
+
+    /**
+     * Prints the ocean.
+     */
+    void print() {
```

Comment:
you can remove this method 
since it is not necessary on edx
or you can return a String method a test that.